-inputs.tf

```diff
@@ -14,7 +14,7 @@ variable "user_name" {
 }
 
 variable "user_home" {
-  description = "HOME path for transfer server user"
+  description = "HOME path for transfer server user. Must start with /"
   type        = string
   default     = "/"
```

Comment:
Can you change the default value from "/" to ""?

Reply:
We also need to change the secret code from 

```hcl
resource "aws_secretsmanager_secret_version" "secret" {
  secret_id     = aws_secretsmanager_secret.secret.id
  secret_string = <<EOF
{
  "Password": "ChangeMe",
  "PublicKey": "${var.ssh_key}",
  "Role": "${aws_iam_role.sftp_transfer_server_user.arn}",
  "HomeDirectory": "/${var.s3_bucket_name}${var.user_home}"
}
```

To

```hcl
resource "aws_secretsmanager_secret_version" "secret" {
  secret_id     = aws_secretsmanager_secret.secret.id
  secret_string = <<EOF
{
  "Password": "ChangeMe",
  "PublicKey": "${var.ssh_key}",
  "Role": "${aws_iam_role.sftp_transfer_server_user.arn}",
  "HomeDirectory": "/${var.s3_bucket_name}/${var.user_home}"
}
```

for maintaining compatibility. It put the code here because as it has not been modified I can't add the comment there.